coreservices/partsrelationshipservice/cd/terraform-connector-consumer/modules/connector_storage/main.tf

```
@@ -0,0 +1,58 @@
+# Retrieve identity information for the current logged-in user
+data "azurerm_client_config" "current" {}
+
+# Retrieve the Key Vault for storing generated identity information and credentials
+data "azurerm_key_vault" "identities" {
+  name                = "cxmtpdc1-${var.environment}-prs-id"
```

```suggestion
  name                = "${var.prefix}-${var.environment}-prs-id"
```
as in other deployments